packages/radix-ui-themes/src/helpers/extract-props.ts

```diff
@@ -47,7 +47,7 @@ function extractProps<
       }
 
       if (propDef.type === 'string') {
-        className = classNames(propDef.className, className);
+        className = classNames(className, propDef.className);
```

Comment:
Sneaking in a consistency improvement to how prop-driven classnames should all appear at the end of the className list